documentation/build.sbt

```diff
@@ -25,11 +25,10 @@ val `json-schema-jvm` = LocalProject("json-schemaJVM")
 val `json-schema-circe-jvm` = LocalProject("json-schema-circeJVM")
 val `json-schema-generic-jvm` = LocalProject("json-schema-genericJVM")
 
-import sbtunidoc.Plugin.UnidocKeys.unidoc
-
 val apiDoc =
   project.in(file("api-doc"))
-    .settings(noPublishSettings ++ `scala 2.11` ++ unidocSettings: _*)
+    .enablePlugins(ScalaUnidocPlugin)
+    .settings(noPublishSettings ++ `scala 2.11`: _*)
```

Comment:
We should try to use Scala 2.12 to generate the API documentation to benefit from the new design/layout.